apps/dashboard/components/batch-payments/sample-table.tsx

```diff
@@ -1,14 +1,13 @@
 import React from "react"
 import Table from "@mui/joy/Table"
-import { Box, Button } from "@mui/joy"
+import { Button } from "@mui/joy"
 import DownloadIcon from "@mui/icons-material/Download"
 
 const SampleCSVTable = () => {
   const createCsvContent = () => {
-    const csvRows = [["username", "dollars", "memo"]]
+    const csvRows = [["username", "amount", "currency", "wallet", "memo"]]
```

Comment:
what is wallet used for?

Reply:
To indicate the wallet type we are using i.e BTC/USD